src/components/SearchForm.js

```diff
@@ -107,7 +106,6 @@ export default class SearchForm extends React.Component {
         this.setState({
             results: results,
             noResults: results.length === 0,
-            queryId: this.state.queryId + 1,
```

Comment:
What happened here? I thought this was needed to get the graph view refreshing, did you work out a way to do it without?

Reply:
This is just to prevent us from re-creating the whole graph if the query the user entered hasn't changed.